### DISCUSSION TITLE
Where to get login code

### DISCUSSION THREAD

**Discussion by kri164** [stalwartlabs/jmap-server#36] [Q&A] [answered]
After jmap server installed I changed admin password, did create new user and opened [url] (https://jmap.example.org/auth) from web browser and got this login form.
Where I get this "Enter code"?

Thank you

![jmap](https://user-images.githubusercontent.com/52274164/220143022-2c545fec-6073-42ff-8d69-ddc21d344efc.png)

**Comment by mdecimus** [comment, marked as answer]
That code will be provided by the OAuth client. For example, if you use the ``stalwart-cli`` CLI to manage the JMAP server and select OAuth authentication, that code will be provided to you.

Please note that Stalwart JMAP does not offer a web interface, to access your messages you need to use a third-party JMAP client.

**Reply by kri164** [reply]
Can you show some CLI command example, how to create or display this code, please?
I cannt find any info in docs.

**Reply by mdecimus** [reply]
Here it is in the docs: https://stalw.art/jmap/manage/overview/

**Reply by kri164** [reply]
It must have been temporary blindness for me not to see it.
Thank you very much.
